others/RESOURCES.md

```diff
@@ -11,3 +11,10 @@ Anything useful or informative that can be found on the internet, including arti
 * [mCoding](https://www.youtube.com/@mCoding) - Python
 * [Luke Smith](https://www.youtube.com/@LukeSmithxyz) - Linux, Shells, Life
 * [Hussein Nassar](https://www.youtube.com/@hnasr) - Backend System Design
+* [Honeypot](https://www.youtube.com/@Honeypotio) - Interesting documentaries about different technologies
+* [Fireship.io](https://www.youtube.com/@Fireship) - JS, TS, Code Reports, 100s
+* [OWASP](https://owasp.org/) - Cybersecurity
+* [Computerphile](https://www.youtube.com/@Computerphile/featured) - Networking, Algorithms, Complex CS topics explained.
+* [sentdex](https://www.youtube.com/@sentdex) - Python, ML
+* [CodeBullet](https://www.youtube.com/@CodeBullet) - Funny videos about recreating games
```

Comment:
recreating what type of games?

Reply:
Maknly online or browser games. But he makes them with AI. For example Donkey Kong.

Reply:
The point of this comment was to make you expand on the description you've provided. 

E.g.
> Funny and insightful videos on the topic of creating online and browser games by utilizing AI

Although, it would be preferable if you actually figure out a way how to shorten the above description

Reply:
Done